packages/react-components/src/organisms/MembersSection.tsx

```diff
@@ -45,12 +45,15 @@ const roleStyles = css({
 });
 
 type MembersSectionProps = {
-  readonly members: TeamMember[];
+  readonly title: string;
```

Comment:
Hmm, this duplicates the `Team Members` in other places. How would you feel about having this optional, defaulting to `Team Members (members.length)`?

Reply:
I thought about it, no strong feelings about it.